site/static/examples/sample-validating-policy.yaml

```
@@ -0,0 +1,22 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: sample-validating-admission-policy
+spec:
+  failurePolicy: Fail
+  matchConstraints:
+    resourceRules:
+    - apiGroups:   ["batch"]
+      apiVersions: ["v1"]
+      operations:  ["CREATE", "UPDATE"]
+      resources:   ["jobs"]
+    - apiGroups:   ["jobset.x-k8s.io"]
+      apiVersions: ["v1alpha2"]
+      operations:  ["CREATE", "UPDATE"]
+      resources:   ["jobsets"]
+  matchConditions:
+    - name: "namespace-scoped-policy"
+      expression: "object.metadata.namespace == 'my-namespace'"
```

```suggestion
      expression: "object.metadata.namespace != 'admin-namespace'"
```
To make it a bit more useful as per https://github.com/kubernetes-sigs/kueue/issues/2119

---

Done and moved validation to the Binding.